PovertyInfo/src/interactiveinfo/GetPovertyInfo.java

```diff
@@ -0,0 +1,30 @@
+package interactiveinfo;
+
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+
+public class GetPovertyInfo {
+
+	public static String getMatch(String county) throws FileNotFoundException {
+		File data = new File("support/povWA.txt");
+		Scanner scnr = new Scanner(data);
+		scnr.nextLine(); // skip first line
+		
+		while(scnr.hasNextLine()) {
+			String line = scnr.nextLine();
+			if ((county + " County").equals(line.substring(193, 238).trim()) ) {
```

Comment:
Could you adjust the substring function to ignore the "county" part and simply check if `county.equals(` etc?